index.html

```diff
@@ -18,42 +18,312 @@
+				</p>
+				<ul class="list-icons">
+					<li>
+						<a href="#"><img src="Works-Images/html2.png"></a>
+					</li>
+					<li>
+						<a href="#"><img src="Works-Images/bootstrap2.png"></a>
+					</li>
+					<li>
+						<a href="#"><img src="Works-Images/Ruby2.png"></a>
+					</li>
+				</ul>
+				<button class="btn" type="button">See Project</button>
+			</div>
+
+			<div class="sec-2 center">
+				<h3 class="h3-light">
+					Profesional Art <br>
+					Printing Data
+				</h3>
+				<p class="p-light">
+					A daily selection of privately <br>
+					personalized reads; no accounts or <br>
+					sign-ups required. Has been the <br>
+					industry's standard.
+				</p>
+				<ul class="list-icons">
+					<li>
+						<a href="#"><img src="Works-Images/html2.png"></a>
+					</li>
+					<li>
+						<a href="#"><img src="Works-Images/bootstrap2.png"></a>
+					</li>
+					<li>
+						<a href="#"><img src="Works-Images/Ruby2.png"></a>
+					</li>
+				</ul>
+				<button class="btn" type="button">See Project</button>
+			</div>
+
+			<div class="sec-2 center">
+				<h3 class="h3-light">
+					Profesional Art <br>
+					Printing Data
+				</h3>
+				<p class="p-light">
+					A daily selection of privately <br>
+					personalized reads; no accounts or <br>
+					sign-ups required. Has been the <br>
+					industry's standard.
+				</p>
+				<ul class="list-icons">
+					<li>
+						<a href="#"><img src="Works-Images/html2.png"></a>
+					</li>
+					<li>
+						<a href="#"><img src="Works-Images/bootstrap2.png"></a>
+					</li>
+					<li>
+						<a href="#"><img src="Works-Images/Ruby2.png"></a>
+					</li>
+				</ul>
+				<button class="btn" type="button">See Project</button>
+			</div>
+
+		</section>
+		<section class="Bottom">
+			<section class="Aboutme">
+				<h2 class="h2-orange">
+					About me
+				</h2>
+				<p class="p-light">
+					Hello I’m a software developer! I can help <br>
+					you build a product , feature or website <br>
+					Look through some of my work and <br>
+					experience! If you like what you see and <br>
+					have a project you need coded, don’t <br>
+					hesitate to contact me.
+				</p>
+				<button type="button" class="btn">Get my resume</button>
+			</section>
+			<section class="Techs">
+				<div class="languages center">
+					<hr>
+
+					<ul>
+						<li class="diamond large symbol ">Languages</li>
+						<ul class="no-style">
+							<li>JavaScript</li>
+							<li>Ruby</li>
+							<li>HTML</li>
+							<li>CSS</li>
+						</ul>
+					</ul>
+
+					<ul>
+						<li class="large symbol">Frameworks</li>
+						<ul class="no-style">
+							<li>React.js</li>
+							<li>Ruby on Rails</li>
+							<li>RSpec</li>
+							<li>Capybara</li>
+						</ul>
+					</ul>
+
+					<ul>
+						<li class="large circle">Skills</li>
 					</ul>
-					
+					<ul class="skill">
+						<li>Database Management</li>
+						<li>Version Control</li>
+						<li>CLI</li>
+						<li>API Design</li>
+						<li>Web Development</li>
+					</ul>
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up_2: please use `ul` and `li` for categories in the "About myself" section. Languages, Frameworks, and Skills should be list items but still, it's not the same as suggested by the previous reviewer :point_up_2: I really appreciate your effort in trying to match your design with the design given 👏. In addition, great job you have used list tags here 👏. What I would like you to improve on is that, as you see the technology categories("Languages", "Frameworks", "Skills") are list items and it is right but their belongings(for example; Languages list contains "Ruby", "Javascript", etc ) should be nested inside them. Let me show you what I would like you to do by my suggestions below :point_down: 

```HTML
<ul>
        <li> Languages
          <ul>
            <li>Javascript</li>
            <li>Ruby</li>
            <li>Html</li>
            <li>Css</li>
          </ul>
        </li>
        <li> Frameworks
          <ul>
            <li>Bootstrap</li>
            <li>Ruby on Rails</li>
            <li>React Js</li>
            <li>Node Js</li>
            <li>Express Js</li>
            <li>Material UI</li>
            <li>Tailwind CSS</li>
          </ul>
        </li>
        <li> Skills
          <ul>
            <li>Codekit</li>
            <li>Github</li>
            <li>Codepen</li>
            <li>Terminal</li>
          </ul>
        </li>
</ul>
```

This is what I would like you to follow because it is **more logical** and it is among the requirements. It is referred to as **Nesting list into another list**. For more information, please read this [documentation](https://www.webucator.com/article/how-to-properly-nest-lists-in-html/) 💡 😃